### PR TITLE
misc(treemap): add GA snippet for new property

### PR DESCRIPTION
**Pull request by connorjclark**
Just tracking page views for now.

ref #11254

### CODE REVIEW

File: lighthouse-treemap/app/index.html [hunk 1]
```diff
@@ -87,17 +87,14 @@
 
   <script src="src/bundled.js"></script>
 
-  <!-- Google Analytics -->
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-16X003MB7V"></script>
```
Comment (patrickhulce) [inline lighthouse-treemap/app/index.html:91]: 😢

File: lighthouse-treemap/app/index.html [hunk 2]
```diff
@@ -87,17 +87,14 @@
...
-      i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
-        (i[r].q = i[r].q || []).push(arguments)
-      }, i[r].l = 1 * new Date(); a = s.createElement(o),
-        m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
-    })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
 
-    ga('create', 'UA-85519014-2', 'auto');
```
Comment (patrickhulce) [inline lighthouse-treemap/app/index.html:99]: can we just change the property descriptor? analytics [still documents this approach](https://developers.google.com/analytics/devguides/collection/analyticsjs) and I'd hate to not follow our own aspirational advice

Comment (connorjclark) [reply lighthouse-treemap/app/index.html:99]: I don't know, and the different prefix in the identifier makes me think twice about doing that.

Comment (patrickhulce) [reply lighthouse-treemap/app/index.html:99]: well this majorly sucks https://support.google.com/analytics/answer/9539598?hl=en 😭

**Review by patrickhulce — APPROVED** [review]

File: lighthouse-treemap/app/index.html (hunk 2 above)
Comment (patrickhulce) [reply lighthouse-treemap/app/index.html:99]: OK, so GA 4 isn't *as bad* as I thought

good news: stopped doing the double script hop, so gtag is the only script loaded to fire GA events
bad news: the script is 140% larger and increases TBT 385% more (granted GA only contributes ~5ms with the original so quintupling only gives you ~25ms, but hey I'm still a little salty 😉 )

**Pull request merged**